### ISSUE TITLE
Entries of Zdnet.fr feed are (almost) never updated

### ISSUE THREAD

**Issue by LaurentGrenet** [nt1m/livemarks#342]
**Feed URL**
https://www.zdnet.fr/feeds/rss/

**Add-on version**
Add-on version:Latest, ie. 3.3

**Describe the bug**
The list of entries is (almost) never updated. I guess this is likely due to a bug in cache management by LiveMarks, since if I deactivate LiveMarks, and put the URL in address bar, I get an up-to-date list of entries. But as soon as I re-activate LiveMarks, putting the URL in address bar gives a list of entries not up-to-date (and consistent with what add-on's bookmark).

I have checked in Firefox Options / Private / Manage site's data, but there is no data nor cookie stored about this website (to be more exact : all data relative to the site, if exist, have already been cleansed by another add-on, "Forget-me-not".

I have writen above "almost never updated", because sometimes, but very very very rarely (once every 10 - 30 days, or even more), without doing anything, entries list is updated. What is the cause of this update ? No idea.

I don't know if this is specific to this feed, but among all feeds I have, this one is the only one with the bug.

Currently, It's not possible to use this feed. I have only maintained it to see if the problem was disappearing, before creating this issue. But it's now several months (> 6 months ?) I have the issue...

To be sure, of course, I tried to delete the feed in Livemarks, close Firefox, reopen it, then re-create the feed. But this doesn't change anything : entries list is still out of date.

**To Reproduce**
1) Create this feed with LiveMarks, going to  https://www.zdnet.fr/actualites/ and clicking on the LiveMarks icon in address bar
First time, most likely is that list of entries will be consistent with what you can see on https://www.zdnet.fr/actualites/

2) Go back the day after. (Zdnet.fr is updated several times a day, but from one day to the day after, we are 100% sure there are new "News")
List of entries in LiveMarks bookmark is still the same than the day before.
Nevertheless, if you go directly to https://www.zdnet.fr/actualites/, you can see new more recent "News" that were not existing the day before, and are not in LiveMarks entries for this feed.

3) Deactivate LiveMarks

4) Open www.Zdnet.fr/feed/rss
Since LiveMarks is deactivated, you can see in top of XML file new entries.

5) Reactivate LiveMarks

6) Re-Open www.Zdnet.fr/feed/rss (or click on LiveMarks icon in address bar, when on https://www.zdnet.fr/actualites/)
In the list (managed by LiveMarks), new entries are not there

**Expected behavior**
To have entries managed by LiveMarks updated each time a new entry is created on the site

**Comment by LaurentGrenet** [comment]
Actually, there were probably an issue **not only** in Livemarks, but maybe also in my Firefox profile.
Because, when I wanted to re-create this feed in LiveMarks... I got an error : Impossible de récupérer le flux
And it was not the case in other environments (eg. on VMs running on Virtualbox)

Then I created a new Firefox profile... in which creation of the feed in LiveMarks is OK.
I let it in place, and in a few days, I'll be able to see if it is automatically updated or not.

Rather than to create a new profile, I would have prefer to be able to purge any reference to Livemarks in my initial profile, then to reload Livemarks. But even after a full de-install of livemarks, then stop/start Firefox, then re-install Livemarks, it comes with all its previous parameters. 
**So my question** : How can we remove all parameters of a de-installed extension, to restart parameterization from scratch when re-installing the extension ?
